From 9274e51a088f3951e66d916ae23be0846269be1a Mon Sep 17 00:00:00 2001
From: Richard Gong <richard.gong@intel.com>
Date: Wed, 12 Feb 2020 12:17:47 -0600
Subject: [PATCH 038/120] firmware: stratix10-svc: add the compatible value for
 intel agilex

commit b1d5e5b785cb4a1580cc77cda07cca69d49aecd6 from
https://github.com/altera-opensource/linux-socfpga.git

[upstream commit c115f65bf0059a666deafd1b24709a7ac562ecbb]

Add the compatible property value so we can reuse Intel Stratix10
Service Layer driver on Intel Agilex SoC platform.

Signed-off-by: Richard Gong <richard.gong@intel.com>
Integrated-by: Abhishek Paliwal <paliwal.abhishek@windriver.com>
---
 drivers/firmware/stratix10-svc.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/firmware/stratix10-svc.c b/drivers/firmware/stratix10-svc.c
index f59fa1ea0c17..a52b26896d7d 100644
--- a/drivers/firmware/stratix10-svc.c
+++ b/drivers/firmware/stratix10-svc.c
@@ -953,6 +953,7 @@ EXPORT_SYMBOL_GPL(stratix10_svc_free_memory);
 
 static const struct of_device_id stratix10_svc_drv_match[] = {
 	{.compatible = "intel,stratix10-svc"},
+	{.compatible = "intel,agilex-svc"},
 	{},
 };
 
-- 
2.31.1

